Dockerfile

```diff
@@ -12,10 +12,32 @@ RUN curl -sL https://deb.nodesource.com/setup_6.x | bash \
     && apt-get install --no-install-recommends -y nodejs \
     && rm -rf /var/lib/apt/lists/*
 
+# Kosmtik with plugins
 RUN npm install -g kosmtik
 
+WORKDIR /usr/lib/node_modules/kosmtik/
```

Comment:
I have to admit I never thought about that. Maybe try to install the plugins globally too? With `npm install -g plugin`? I'll have to try it. Meanwhile, could you please open a issue about that at Kosmtik too?